lms/djangoapps/branding/api.py

```diff
@@ -0,0 +1,135 @@
+"""Edx footer API
+"""
+import logging
+from util.json_request import JsonResponse
+from django.conf import settings
+from django.utils.translation import ugettext as _
+from microsite_configuration import microsite
+
+log = logging.getLogger("edx.footer")
+
+
+def get_footer():
+    """ Get the footer links json
+
+    Returns:
+        Dict of footer links
+    """
+    site_name = microsite.get_value('SITE_NAME', settings.SITE_NAME)
+    context = dict()
+    context["copy_right"] = copy_right()
+    context["heading"] = heading()
+    context["logo_img"] = "{site_name}/sites/all/themes/atedx/images/edx-logo-footer.png".format(site_name=site_name)
+    context["social_links"] = social_links()
+    context["about_links"] = about_edx_link(site_name)
+
+    return JsonResponse({"footer": context}, 200)
+
+
+def copy_right():
+    """ Returns the copy rights text
+    """
+    data = _("(c) 2015 edX Inc. EdX, Open edX, and the edX and Open edX logos "
```

Comment:
This should serve both the open source and edx.org versions of the copyright.  See my comment below for more info.